### ISSUE TITLE
Rotation offset

### ISSUE THREAD

**Issue by CyleJS** [mclemente/about-face#29]
Some art packs have top down tokens at diagonals instead of facing down. In these cases, tokens face the wrong way once they start moving. It would be nice to have a rotation offset angle that we could enter so the module would work correctly in those cases.

The rotation offset in Foundry default works for initial token placement, but gets reset as soon as the token moves for the first time.

**Comment by tprussak** [comment]
I see the change to set the flag with the hotkey, but then the hotkey seems to have been removed in the next change. Is there some other way to set the rotation offset flag?

**Comment by mclemente** [comment]
That implementation had its share of problems if you selected multiple tokens at once, so it didn't make it into a release. You should set the rotation on the Prototype Token configuration and it'll become the offset flag once the token is placed.

**Comment by tprussak** [comment]
I tried that, it gets reset after moving the token. I tested it again on with just About Face 3.17, lib - Color Settings 2.8.4, and libWrapper 1.12.7.0, on Foundry V9 Build 269, Dnd5e System 1.6.3, no other modules active. Setting the rotation on the prototype token, then dragging the token out, it will be correctly rotated at first (including with respect to the direction indicator), but as soon as it moves the rotation resets.
